WeakAuras/BuffTrigger2.lua

```diff
@@ -967,6 +967,14 @@ local function TriggerInfoApplies(triggerInfo, unit)
     return false
   end
 
+  if triggerInfo.arenaSpec and unit:sub(1, 5) == "arena" then
+    -- GetArenaOpponentSpec doesn't use unit ids!
+    local i = tonumber(unit:sub(6))
+    if not triggerInfo.arenaSpec[GetArenaOpponentSpec(i)] then
```

Comment:
GetArenaOpponentSpec is nil on bcc & classic

Reply:
That code path can never be encountered on classic/bcc

Reply:
Are you sure it can't be encounter if you import an aura with that option enable from Retail to BCC ?

Reply:
Yes it checks if the unit id matches arena two lines above, and that unit id comes from the game.